GUI/Quiz/quiz.py

```diff
@@ -23,7 +23,7 @@
     10: ["To avoid being blinded by the headlights of another vehicle coming towards you what should you do?", 'Look to the left of the road', "Look to the centre of the road", "Wear sunglasses that have sufficient strength", 'Look to the right side of the road', 'Look to the left of the road',1],
     }
 
-class Quiz:
+class QuizStarter:
```

Comment:
Ms Bsisou had two classes in her code. One was called QuizStarter (the first class made in Parts 1 and 2) and then the Quiz class, which contains the radio buttons.